sdk/batch/Microsoft.Azure.Batch/Tools/ObjectModelCodeGenerator/Spec/CloudJob.json

```diff
@@ -112,7 +112,7 @@
           "Key": {
             "Type": "JobReleaseTask",
             "Name": "JobReleaseTask",
-            "SummaryComment": "The Job Release task. The Batch service runs the Job Release task when the job ends, on each compute node where any task of the job has run.",
+            "SummaryComment": "The Job Release Task runs when the Job ends, because of one of the following: The user calls the Terminate Job API, or the Delete Job API while the Job is still active, the Job's maximum wall clock time constraint is reached, and the Job is still active, or the Job's Job Manager Task completed, and the Job is configured to terminate when the Job Manager completes. The Job Release Task runs on each Node where Tasks of the Job have run and the Job Preparation Task ran and completed. If you reimage a Node after it has run the Job Preparation Task, and the Job ends without any further Tasks of the Job running on that Node (and hence the Job Preparation Task does not re-run), then the Job Release Task does not run on that Compute Node. If a Node reboots while the Job Release Task is still running, the Job Release Task runs again when the Compute Node starts up. The Job is not marked as complete until all Job Release Tasks have completed. The Job Release Task runs in the background. It does not occupy a scheduling slot; that is, it does not count towards the taskSlotsPerNode limit specified on the Pool.",
```

Comment:
This is difficult to follow. Should we maybe just link to some documentation?

Reply:
This is from the swagger definition. There are some problems with hardlinking external documentation, so generally this is meant as a summary and not full clarity.